docs/submitting-applications.md

```diff
@@ -192,7 +192,7 @@ debugging information by running `spark-submit` with the `--verbose` option.
 
 # Advanced Dependency Management
 When using `spark-submit`, the application jar along with any jars included with the `--jars` option
-will be automatically transferred to the cluster. URLs supplied after `--jars` must be separated by commas. That list is included in the driver and executor classpaths. Directory expansion does not work with `--jars`.
+will be automatically transferred to the cluster. URLs supplied after `--jars` must be separated by commas. That list won't be included in the driver and executor classpaths. You need to explicitly add them using `spark.driver.extraClassPath` and `spark.executor.extraClassPath`. Directory expansion does not work with `--jars`.
```

Comment:
?? do you mean adding `--jars` it self doesn't work for your case? Please elaborate it why and how in the PR description and some pointers for the codes if possible.